### ISSUE TITLE
 remove stop words in word cloud

### ISSUE THREAD

**Issue by TobiasLee** [acl-org/arr-health#8]
Nice job for the word cloud visualization of abstract and title!
There are some words like pronouns (like we) that can be removed to leave spaces for more informative topic words.

**Comment by SebastinSanty** [comment]
@TobiasLee Thank you for pointing this out. I did notice it and thought those stopwords might be interesting in seeing how people write (title has 'A' and abstract has 'We'). But yes, for seeing the trends in NLP term use, it is better to remove stopwords as you suggest. I will make this change.

**Comment by TobiasLee** [comment]
Yeah, thanks for taking this consideration and I am looking forward to seeing a new cloud.
